webapp-django/crashstats/crashstats/sentry.py

```diff
@@ -0,0 +1,327 @@
+"""Provides a before_send callable to sanitize Sentry events."""
```

Comment:
Also, since this applies to the whole code base and we're importing this from settings, this shouldn't be in an app and should instead be in `webapp-django/crashstats/sentry.py`. That prevents potential bootstrap and circular import problems down the road.

Also, since it doesn't contain Sentry itself, but rather contains some utility things we wrote to use with Sentry, I'd call it `sentrylib.py` or `sentryutils.py`.